common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -101,5 +140,10 @@
     if (baseFontSize && parseInt(baseFontSize, 10) !== 16) {
         documentElement.style.fontSize = baseFontSize
     }
+
+    if (hideReaderRevenue()) {
+        docClass += ' hide-reader-revenue';
```

Comment:
Well - I thought about it and think, as a developer I'd go with show instead of hiding, but I believe it would be better for the business to reverse that and show the messages by default and hide them, as part of the progressive enhancement.

Could you agree on that?